doc/tutorials/rdt_configuration.rst

```diff
@@ -0,0 +1,194 @@
+
+#. Set up CLOS in the VM config. Please follow `RDT detection and resource capabilites` to identify the MAX CLOS that can be used. In ACRN we a value between 0 to **the lowest common MAX CLOS** among all the RDT resources to avoid resource misconfigurations. We will take SOS on sharing mode as an example. Its configuration data can be found at ``hypervisor/arch/x86/configs/vm_config.c``
+
+   .. code-block:: none
+      :emphasize-lines: 6
+
+      struct acrn_vm_config vm_configs[CONFIG_MAX_VM_NUM] __aligned(PAGE_SIZE) = {
+              {
+                      .type = SOS_VM,
+                      .name = SOS_VM_CONFIG_NAME,
+                      .guest_flags = 0UL,
+                      .clos = 1,
+                      .memory = {
+                              .start_hpa = 0x0UL,
+                              .size = CONFIG_SOS_RAM_SIZE,
+                      },
+                      .os_config = {
+                              .name = SOS_VM_CONFIG_OS_NAME,
+                      },
+              },
+      };
+
+.. note::
+   In ACRN, Lower CLOS always means higher priority (clos 0 > clos 1 > clos 2>...clos n). So care should be taken to program the VMs CLOS accordingly.
+
+#. Careful consideration should be made in assigning VCPU affinity. In cache isolation configuration, not only need to isolate CAT-capable caches, but need to isolate lower-level caches as well. In the following example, logical processor #0 and #2 share L1 and L2 caches. In this case, do not assign LP #0 and LP #2 to different VMs that need to do cache isolation. Assign LP #1 and LP #3 with similar consideration.
+
+   .. code-block:: none
+      :emphasize-lines: 3
+
+      # lstopo-no-graphics -v
+      Package L#0 (P#0 CPUVendor=GenuineIntel CPUFamilyNumber=6 CPUModelNumber=142)
+        L3Cache L#0 (size=3072KB linesize=64 ways=12 Inclusive=1)
+          L2Cache L#0 (size=256KB linesize=64 ways=4 Inclusive=0)
+            L1dCache L#0 (size=32KB linesize=64 ways=8 Inclusive=0)
+              L1iCache L#0 (size=32KB linesize=64 ways=8 Inclusive=0)
+                Core L#0 (P#0)
+                  PU L#0 (P#0)
+                  PU L#1 (P#2)
+          L2Cache L#1 (size=256KB linesize=64 ways=4 Inclusive=0)
+            L1dCache L#1 (size=32KB linesize=64 ways=8 Inclusive=0)
+              L1iCache L#1 (size=32KB linesize=64 ways=8 Inclusive=0)
+                Core L#1 (P#1)
+                  PU L#2 (P#1)
+                  PU L#3 (P#3)
+
+#. Similary bandwidth control is per-core (not per LP), so max delay values of per-LP CLOS is applied to the core. If HT is turned on, don’t place high priority threads on sibling LP running lower priority threads.
+
+#. Build the ACRN hypervisor and copy the artifact ``acrn.efi`` to the
```

Comment:
We have had problems in the past with people building and updating the `acrn.efi` hypervisor but not the user-space tools (esp. `acrn-dm`). While we do not want to add full instructions on how to do that, let's at least put in a note to make sure people understand this needs to be done too.